thunder/tests/test_ops.py

```diff
@@ -239,18 +239,60 @@ def foo():
         tfoo()
 
 
-def test_setitem():
-    def fn(a):
-        a[:3] = 2
+@pytest.mark.parametrize("requires_grad", (True, False))
+def test_setitem(requires_grad):
```

Comment:
Have filed an issue for opinfo based testing - https://github.com/Lightning-AI/lightning-thunder/issues/1357

Reply:
Thank you!